src/Umbraco.Web/Mvc/UmbracoRequireHttpsAttribute.cs

```diff
@@ -29,7 +29,7 @@ protected override void HandleNonHttpsRequest(AuthorizationContext filterContext
         public override void OnAuthorization(AuthorizationContext filterContext)
         {
             // If umbracoSSL is set, let base method handle checking for HTTPS.  Otherwise, we don't care.
-            if (Current.Configs.Global().UseHttps)
+            if (/*Current.Configs.Global().UseHttps*/ false)
```

Comment:
This is part of `Umbraco.Web`. Most of this class is already migrated (e.g the constructor). So this will never be executes and is mostly just to allow tests to pass

Reply:
Just to clarify, the whole class will be deleted in future?